libs/core/src/lib/wizard/wizard-step/wizard-step.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="fd-wizard__step-wrapper">
+    <a
+        class="fd-wizard__step-container"
+        [attr.aria-label]="ariaLabel"
+        tabindex="0"
+        (click)="stepContainerKeypress()"
+        (keypress)="stepContainerKeypress($event)"
+    >
+        <ng-content select="fd-wizard-step-indicator"></ng-content>
+        <div class="fd-wizard__label-container" [ngClass]="optionalText ? 'fd-wizard__label-container--optional' : ''">
+            <div #wizardLabel>
+                <span class="fd-wizard__label">{{ label }}</span>
+                <span *ngIf="optionalText" class="fd-wizard__optional-text">{{ optionalText }}</span>
+            </div>
+        </div>
+    </a>
+    <span
+        *ngIf="!finalStep"
+        class="fd-wizard__connector"
+        [ngClass]="{
```

Comment:
[class.fd-wizard__connector--active]="status === 'completed'"
[class.fd-wizard__connector--branching]="branching"

Reply:
ngClass directive is intended for when you need to add multiple classes, what is the advantage here?

Reply:
`[ngClass]` is really slow, `[class.]` notation faster

Reply:
all of this code of [[ngClass]](https://github.com/angular/angular/blob/10.1.5/packages/common/src/directives/ng_class.ts#L11-L160) vs [class] that Ivy compile into native JS calls